src/stories/Quiz/Component/QuestionTypes/QuestionTypeDecorator.tsx

```diff
@@ -0,0 +1,22 @@
+import { Question } from '@constructor-io/constructorio-client-javascript/lib/types';
+import QuizContext from '../../../../components/CioQuiz/context';
+import { getMockContextValue } from '../../tests/mocks';
+
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+if (process.env.NODE_ENV !== 'production') {
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  const axe = require('@axe-core/react');
```

Comment:
I don't know if this is possible but is there a way to run this similar to lint or tests from CLI? Right now this outputs to console and I think it's very easy to miss. Again, I don't know if this is possible or not, but would be very nice

Reply:
The only way I know of would be to use jest-axe + react testing library and write unit tests for accessibility. 

something like that. 

```
import React from 'react';
import { axe } from 'jest-axe';
import { render } from '@testing-library/react';
import MyComponent from './MyComponent';
it('should not have basic accessibility issues', async () => {
  const { container } = render(<Component />);
  const results = await axe(container);
  expect(results).toHaveNoViolations();    
});

```

Reply:
That looks good to me. Can you create a separate story for that?